resources/yki/localisations/en.json

```
@@ -22,6 +22,15 @@
   "common.testCentre": "Test centre",
   "common.testDay": "Test day",
   "common.testFee": "Test fee",
+  "email.cancel_registration.subject": "Registration for YKI test cancelled",
+  "email.cancel_registration.content.line1": "Your registration for the following YKI test has been cancelled",
+  "email.cancel_registration.content.line2": "You test fee will not be refunded automatically.",
```

Mallipohjiin oli jäänyt pieni typo:

```suggestion
  "email.cancel_registration.content.line2": "Your test fee will not be refunded automatically.",
```